addOns/pscanrulesAlpha/src/main/java/org/zaproxy/zap/extension/pscanrulesAlpha/Base64Disclosure.java

```diff
@@ -198,54 +193,44 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
-                                            + "] ");
-                            if (noDigitInString)
-                                log.trace(
-                                        "The candidate Base64 has no digit characters, and the the probability of this occurring for a string of this length is "
-                                                + (probabilityOfNoDigitInString * 100)
-                                                + "%. The threshold is "
-                                                + (probabilityThreshold * 100)
-                                                + "%");
-                            if (noAlphaInString)
-                                log.trace(
-                                        "The candidate Base64 has no alphabetic characters, and the the probability of this occurring for a string of this length is "
-                                                + (probabilityOfNoAlphaInString * 100)
-                                                + "%. The threshold is "
-                                                + (probabilityThreshold * 100)
-                                                + "%");
-                            // if (noOtherInString)
-                            //	log.trace("The candidate Base64 has no 'other' characters, and the
-                            // the probability of this occurring for a string of this length is "+
-                            // (probabilityOfNoOtherInString * 100) + "%. The threshold is "+
-                            // (probabilityThreshold *100)+ "%");
-                            if (noLowerInString)
-                                log.trace(
-                                        "The candidate Base64 has no lowercase characters, and the the probability of this occurring for a string of this length is "
-                                                + (probabilityOfNoLowerInString * 100)
-                                                + "%. The threshold is "
-                                                + (probabilityThreshold * 100)
-                                                + "%");
-                            if (noUpperInString)
-                                log.trace(
-                                        "The candidate Base64 has no uppercase characters, and the the probability of this occurring for a string of this length is "
-                                                + (probabilityOfNoUpperInString * 100)
-                                                + "%. The threshold is "
-                                                + (probabilityThreshold * 100)
-                                                + "%");
-                        }
+                                    "The candidate Base64 has no digit characters, and the the probability of this occurring for a string of this length is {}%. The threshold is {}%",
+                                    (probabilityOfNoDigitInString * 100),
```

Comment:
The parentheses can be removed now (same for other log statements).

Reply:
There was also some in the commented statement.

Reply:
Fixed